run-tests.sh

```diff
@@ -24,7 +24,7 @@
 
 
 pep257 invenio_rest && \
-isort -rc -c -df **/*.py && \
+#isort -rc -c -df **/*.py && \
```

Comment:
You can fix the isort problem by adding ``flask_limiter and`` and ``flask_cors`` (because the exists in requirements-devel.txt) modules in ``.editorconfig`` (see fix in pidstore by @krzysztof)